### PR TITLE
Command line flags for google_speech options

### PR DESCRIPTION
**Pull request by chetan3clogic**
- Now, there is no need to set a channel variable for google_speech_option.

- New Command: 

```bash
uuid_google_transcribe <uuid> start <lang-code> [interim] [single-utterence](bool) [seperate-recognition](bool) [max-alternatives](int) [profinity-filter](bool) [word-time](bool) [punctuation](bool) [model](string) [enhanced](bool) [hints](word seperated by , and no spaces)
```
Example:
```bash
uuid_google_transcribe a768fc24-992f-4696-aecb-3e9c11409902 start en-US interim true true 5 true true true command_and_search true yes,no,hello
```

- Also now sample_rate is dynamic for each call
- Logging also added
image

### CODE REVIEW

File: modules/mod_google_transcribe/google_glue.cpp [hunk 1]
```diff
@@ -45,13 +47,18 @@ class GStreamer {
...
       streaming_config->set_single_utterance(true);
     }
 
 		config->set_language_code(lang);
-  	config->set_sample_rate_hertz(8000);
+    switch_log_printf(SWITCH_CHANNEL_SESSION_LOG(m_session), SWITCH_LOG_DEBUG, "transcribe language %s \n", lang);
+    
+    int sample_rate = atoi(switch_channel_get_variable(channel, "read_rate"));
+  	config->set_sample_rate_hertz(sample_rate);
+    switch_log_printf(SWITCH_CHANNEL_SESSION_LOG(m_session), SWITCH_LOG_DEBUG, "sample rate %d \n", sample_rate);
```
Comment (anshumanr) [inline modules/mod_google_transcribe/google_glue.cpp:60]: is `sample_rate` set internally by FS? or do we have to set this before the call to this function?

**Comment by anshumanr** [comment]
i added some changes as per discussion [here](https://github.com/davehorton/drachtio-freeswitch-modules/issues/52). But don't know how to compile & test. Could you please check & test. Will discuss.

File: modules/mod_google_transcribe/google_glue.cpp (hunk 1 above)
Comment (chetan3clogic) [reply modules/mod_google_transcribe/google_glue.cpp:60]: it is done by fs itself

**Comment by chetan3clogic** [comment]
- Now, there is no need to set a channel variable for google_speech_option.

- New Command: 

```bash
uuid_google_transcribe2 <uuid> start <lang-code> [interim] [single-utterence](bool) [seperate-recognition](bool) [max-alternatives](int) [profinity-filter](bool) [word-time](bool) [punctuation](bool) [model](string) [enhanced](bool) [hints](word seperated by , and no spaces)
```
Example:
```bash
uuid_google_transcribe2 a768fc24-992f-4696-aecb-3e9c11409902 start en-US interim true true 5 true true true command_and_search true yes,no,hello
```

- Also now sample_rate is dynamic for each call
- Logging also added
image

**Comment by chetan3clogic** [comment]
> i added some changes as per discussion [here](https://github.com/davehorton/drachtio-freeswitch-modules/issues/52). But don't know how to compile & test. Could you please check & test. Will discuss.

I have made some changes (for compilation) and tested uuid_google_transcribe2 its working fine

**Review by anshumanr — APPROVED** [review]

**Pull request merged**